.github/services/redis/redis_with_cluster_tls/action.yml

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: redis_with_cluster_tls
+description: 'Behavior test for redis with cluster tls'
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v4
+
+    - name: Setup Redis Cluster with TLS
+      shell: bash
+      working-directory: fixtures/redis
+      run: |
+
+        # Install the CA in the system
+
+        sudo cp ssl/ca.crt /usr/local/share/ca-certificates
+        sudo update-ca-certificates
+
+        docker compose -f docker-compose-redis-cluster-tls.yml up -d --wait
+
+    - name: Setup
+      shell: bash
+      run: |
+        cat << EOF >> $GITHUB_ENV
+        OPENDAL_REDIS_CLUSTER_ENDPOINTS=rediss://127.0.0.1:6380/,rediss://127.0.0.1:6381/,rediss://127.0.0.1:6382/,rediss://127.0.0.1:6383/,rediss://127.0.0.1:6384/,rediss://127.0.0.1:6385/
+        OPENDAL_REDIS_ROOT=/test/opendal
```

Comment:
you need add `OPENDAL_REDIS_PASSWORD=opendal`

Reply:
thank you very much!